alerts/recommended_alerts_ARM/ContainerCPUPercentage.bicep

```diff
@@ -0,0 +1,123 @@
+@description('Name of the alert')
```

Comment:
Thanks, @carlin-q-scott  for making the PR to add the bicep template support for the recommended alerts.  Did you test these bicep templates works as expected?

Reply:
I'm using these ones in my deployment without issue:

- NodeCPUPercentage.bicep
- NodeDiskUsagePercentage.bicep
- NodeWorkingSetMemoryPercentage.bicep
- PodsInFailedState.bicep
- RestartingContainerCount.bicep
- StaleJobsCount.bicep

I did not verify the other ones, but these are barely modified from what was generated using `az bicep decompile`.